Add Arthurtilly to the CODEOWNERS file

Reonu opened the pull request.

Reonu committed:
Add Arthurtilly to the CODEOWNERS file

arthurtilly reviewed (changes requested):
dont trust this guy

gheskett commented:
I'll come back to this later when I have time, but I feel like there's gotta be a way to make the CODEOWNERS not only look at the bottom entry.

gheskett commented on CODEOWNERS:
```diff
@@ -5,6 +5,7 @@
 *       @Reonu
 *       @thecozies
 *       @gheskett
+*       @arthurtilly
```
Try this actually. I think the issue is that we're re-establishing a global CODEOWNER every time, but by adding them as part of the same line, it should allow everyone to share the role of a global CODEOWNER...I think.

```suggestion
*       @Reonu @thecozies @gheskett @arthurtilly
```

gheskett committed:
Update CODEOWNERS file to list each global owner on the same line

This should (allegedly) fix the issues of only the last entry being treated as an owner for reviews.

gheskett committed:
Remove thecozies from CODEOWNERS file :(

gheskett reviewed (approved):
This is now ready to go. (Please squash this one.)

Reonu commented:
Closing since Arthur doesn't trust that guy
